includes/admin/downloads/metabox.php

```diff
@@ -813,40 +842,45 @@ function edd_render_file_row( $key = '', $args = array(), $post_id, $index ) {
 	</div>
 
 	<div class="edd-repeatable-row-standard-fields<?php echo $variable_class; ?>">
-		<div class="edd-file-name">
-			<span class="edd-repeatable-row-setting-label"><?php _e( 'File Name', 'easy-digital-downloads' ); ?></span>
+		<div class="edd-form-group edd-file-name">
+			<label for="edd_download_files-<?php echo esc_attr( $key ); ?>-name" class="edd-form-group__label edd-repeatable-row-setting-label"><?php esc_html_e( 'File Name', 'easy-digital-downloads' ); ?></label>
+			<div class="edd-form-group__control">
 			<input type="hidden" name="edd_download_files[<?php echo absint( $key ); ?>][attachment_id]" class="edd_repeatable_attachment_id_field" value="<?php echo esc_attr( absint( $args['attachment_id'] ) ); ?>"/>
 			<input type="hidden" name="edd_download_files[<?php echo absint( $key ); ?>][thumbnail_size]" class="edd_repeatable_thumbnail_size_field" value="<?php echo esc_attr( $args['thumbnail_size'] ); ?>"/>
 			<?php echo EDD()->html->text( array(
 				'name'        => 'edd_download_files[' . $key . '][name]',
+				'id'          => 'edd_download_files-' . $key . '-name',
 				'value'       => $args['name'],
 				'placeholder' => __( 'My Neat File', 'easy-digital-downloads' ),
-				'class'       => 'edd_repeatable_name_field large-text'
+				'class'       => 'edd_repeatable_name_field large-text',
 			) ); ?>
+			</div>
 		</div>
 
-		<div class="edd-file-url">
-			<span class="edd-repeatable-row-setting-label"><?php _e( 'File URL', 'easy-digital-downloads' ); ?></span>
-			<div class="edd_repeatable_upload_field_container">
+		<div class="edd-form-group edd-file-url">
+			<label for="edd_download_files-<?php echo esc_attr( $key ); ?>-file" class="edd-form-group__label edd-repeatable-row-setting-label"><?php esc_html_e( 'File URL', 'easy-digital-downloads' ); ?></label>
+			<div class="edd-form-group__control edd_repeatable_upload_field_container">
 				<?php echo EDD()->html->text( array(
 					'name'        => 'edd_download_files[' . $key . '][file]',
+					'id'          => 'edd_download_files-' . $key . '-file',
 					'value'       => $args['file'],
 					'placeholder' => __( 'Enter, upload, choose from Media Library', 'easy-digital-downloads' ),
-					'class'       => 'edd_repeatable_upload_field edd_upload_field large-text'
+					'class'       => 'edd_repeatable_upload_field edd_upload_field large-text',
 				) ); ?>
 
 				<span class="edd_upload_file">
-					<a href="#" data-uploader-title="<?php _e( 'Select Files', 'easy-digital-downloads' ); ?>" data-uploader-button-text="<?php _e( 'Select', 'easy-digital-downloads' ); ?>" class="edd_upload_file_button" onclick="return false;">
+					<button data-uploader-title="<?php esc_html_e( 'Select Files', 'easy-digital-downloads' ); ?>" data-uploader-button-text="<?php esc_attr_e( 'Select', 'easy-digital-downloads' ); ?>" class="edd_upload_file_button" onclick="return false;">
```

Comment:
```suggestion
					<button data-uploader-title="<?php esc_attr_e( 'Select Files', 'easy-digital-downloads' ); ?>" data-uploader-button-text="<?php esc_attr_e( 'Select', 'easy-digital-downloads' ); ?>" class="edd_upload_file_button" onclick="return false;">
```

`esc_attr` here because it's inside an attribute. :)